files/hipchat.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+require "#{File.dirname(__FILE__)}/base"
+
+class Hipchat < BaseHandler
+  def handle
+    timeout_and_retry do
+      case @event['check']['status'].to_i
```

Comment:
this line changed, but this still wasn't addressed.  re commenting: 

@solarkennedy  wrote: 
> Can you switch on @event['action'] instead? Switching based on the status was an original mistake we made and I don't want to see it happen again. (see #49)